src/services/user/services/RegistrationLink.js

```diff
@@ -0,0 +1,73 @@
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+
+const { SC_SHORT_TITLE, SC_DOMAIN } = require('../../../../config/globals');
+
+const mailContent = (firstName, lastName, registrationLink) => {
+	const mail = {
+		subject: `Einladung für die Nutzung der ${SC_SHORT_TITLE}!`,
+		content: {
+			text: `Einladung in die ${SC_SHORT_TITLE}
+    Hallo ${firstName} ${lastName}!
+    \\nDu wurdest eingeladen, der ${SC_SHORT_TITLE} beizutreten,
+    \\nbitte vervollständige deine Registrierung unter folgendem Link: ${registrationLink}
+    \\nViel Spaß und einen guten Start wünscht dir dein ${SC_SHORT_TITLE}-Team`,
+			html: '',
+		},
+	};
+	return mail;
+};
+
+const { userModel } = require('../model');
+
+const getCurrentUserInfo = (id) => userModel.findById(id)
+	.select(['schoolId', 'email', 'firstName', 'lastName'])
+	.populate('roles')
+	.lean()
+	.exec();
+
+class RegistrationLink {
+	async create(data, params) {
+		try {
+			const { userIds } = data;
+
+			for (const userId of userIds) {
+				// get user from ids in query
+				const user = await Promise.resolve(getCurrentUserInfo(userId));
+				if (!user.roles || user.roles.length > 1) {
+					throw new BadRequest('Roles must be exactly of length one if generateRegistrationLink=true is set.');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 121. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5448725)